src/scenes/Document/scenes/Policy/components/Expiration/Expiration.test.js

```diff
@@ -10,7 +10,12 @@ afterEach(cleanup);
 describe('Expiration', () => {
   test('Renders a toggled off expiration', () => {
     const { container, queryByRole } = render(
-      <Expiration policy={new Virtru.PolicyBuilder().build()} policyChange={() => {}} />,
+      <Expiration
+        policy={new Virtru.PolicyBuilder().build()}
+        policyChange={(change) => () => (e) => {
+          e && e.preventDefault();
+        }}
```

Comment:
@dmihalcik-virtru how did you find this bug?

Reply:
I'm not sure the details of how I did this, but if you revert the change you get an warning message cluttering up `npm test`: 

```
    console.error
      Warning: You provided a `checked` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultChecked`. Otherwise, set either `onChange` or `readOnly`.
          at input
          at div
          at Toggle (/Users/dmihalcik/Documents/GitHub/protect-and-track/src/scenes/Document/scenes/Policy/components/Toggle/Toggle.js:4:19)
          at div
          at SectionHeader (/Users/dmihalcik/Documents/GitHub/protect-and-track/src/scenes/Document/scenes/Policy/components/SectionHeader/SectionHeader.js:4:33)
          at div
          at Expiration (/Users/dmihalcik/Documents/GitHub/protect-and-track/src/scenes/Document/scenes/Policy/components/Expiration/Expiration.js:10:23)

      at printWarning (node_modules/react-dom/cjs/react-dom.development.js:86:30)
      at error (node_modules/react-dom/cjs/react-dom.development.js:60:7)
      at checkControlledValueProps (node_modules/react-dom/cjs/react-dom.development.js:1596:7)
      at initWrapperState (node_modules/react-dom/cjs/react-dom.development.js:1775:5)
      at setInitialProperties (node_modules/react-dom/cjs/react-dom.development.js:9888:7)
      at finalizeInitialChildren (node_modules/react-dom/cjs/react-dom.development.js:10950:3)
      at completeWork (node_modules/react-dom/cjs/react-dom.development.js:22193:17)
      at completeUnitOfWork (node_modules/react-dom/cjs/react-dom.development.js:26593:16)
```